packages/internals/src/index.ts

```diff
@@ -36,7 +36,6 @@ export { getGenerator, getGenerators } from './get-generators/getGenerators'
 export { getPackedPackage } from './getPackedPackage'
 export { highlightDatamodel, highlightSql, highlightTS } from './highlight/highlight'
 export type { IntrospectionSchemaVersion, IntrospectionWarnings } from './IntrospectionEngine'
```

Comment:
We can freely remove these exports, right?

Reply:
We need to ping the team that consumes the IntrospectionEngine but yes we should remove this along with the code later

Reply:
Happened in https://prisma-company.slack.com/archives/C01UR43BV6W/p1674224135195389